### ISSUE TITLE
clarify representation of milestones

### ISSUE THREAD

**Issue by kshawkin** [kshawkin/Best-Practices-for-TEI-in-Libraries#4]
For a line if asterisks between paragraphs, we currently say to use `<ab type="typography">******</ab>`.  I think we chose this for ease of rendering a sequence of Unicode characters that's included as content of this element.  We should explain this rationale for choosing `<ab>` (over `<milestone>` or `<space>`, which are more conventional choices).

Furthermore, we should add @style to the example:

```
<ab type="typography" style="text-align: center">******</ab>
```

**Comment by kshawkin** [comment]
Check that we say what to do if you encounter this at a division (`<div>`) boundary.

**Comment by sydb** [comment]
But what we say how to encode this at `<div>` boundary does not effect this ticket. I'm inclined to say “yes”, we should add prose explaining.

**Comment by kshawkin** [comment]
I think Syd is right that what to do about a `<div>` boundary is a separate issue.  In fact, I now remember that [the Best Practices already say](http://www.tei-c.org/SIG/Libraries/teiinlibraries/main-driver.html#index.xml-body.1_div.3_div.6) that when a `<pb>` occurs at a div boundary, we should always include it within a `<div>` for ease of processing.  We probably should be consistent for what we say these milestones.

But on the question originally raised in this issue, Martin Mueller suggested that `<pc>` would be more appropriate than `<ab>`.  He agreed to post to TEI-L to explain that we are looking for a non-empty element for this purpose (ruling out `<milestone>` and `<space>`) and ask for people's thoughts on using `<pc>`.  We'll return to this once we have some thoughts.

**Comment by PFSchaffner** [comment]
`<pc>` is semantically appropriate but I suspect would be practically infeasible, since it is a word/chunk level element that can only appear where plain text can appear, whereas pause-lines or almost-divs, or whatever you want to call these indications of a vague break, occur often in places that are more likely to require an element sibling to the elements on either side (`<p>` or `<lg>` for example). Which I think leaves only `<ab>` and `<metamark>` as practical options, assuming that you want to capture the line of whatevers as literals, and capture them within the matrix within which the sibling elements float (typically a `<div>` but possibly also `<q>` `<floatingText>` etc.) I.e., whatever you choose has to be able to appear directly in `<div>` and preferably also in `<q(uote)>` and `<sp>` and `<floatingText>` and probably other places I haven't thought of.  As for capturing them as literals, we should bear in mind that 'line of asterisks' is only one of many styles, others being dashes, horizontal rules, dots, alternating dots and equals, etc.

**Comment by kshawkin** [comment]
Following discussion during monthly call on 2016-04-04, Martin Mueller posted to TEI-L to ask for input: https://listserv.brown.edu/archives/cgi-bin/wa?A2=TEI-L;96293b6.1604 .

**Comment by kshawkin** [comment]
As discussed during today's monthly meeting, BPTL will recommend recommend putting all milestone-type elements in the lowest-level `<div>`.  However, if a user diverges from this practice, they should document in `<editorialDecl>`. So if we decide on an empty element as the solution for this, we should address placement with respect to `<div>` elements.

**Comment by kshawkin** [comment]
Decided during today's monthly meeting that @emylonas will investigate what P5 says and summarize in a comment here.  Then we'll reconsider.

**Comment by kshawkin** [comment]
For the record, Martin Mueller wrote on teilib-l on 2016-05-17:

> As for typographical milestones, it is the case that they are often ornamental. But they are first of all separators, though of a weak kind. If a div is like a colon, a typographical milestone is like a comma, but there is no clear hierarchy. On the other hand, you never find such phenomena as merely decorative. They always carry the message that what comes after them is in some fashion different from what came before. They may be more like mdashes, indeterminate and anti-hierarchical articulators of text, but articulators nonetheless. The question about what to do with them in an XML environment points to their nature: XML is very binary, but these dividers try to have it both ways.  I'd like to keep alive the `<pc>` option, perhaps as `<pc. unit='discourse'>`, becaue it may be least abusive option. It puts them "in" the text, where they belong, but it doesn't turn them into the kind of content you'd expect to find inside `<ab>`.  The `<pc>` element is a relatively recent addition to the tag set, and it may be worth having a discussion about whether acknowledging a weak discursive "punctuation" is a useful extension of a punctuation element or a form of tag abuse.

**Comment by emylonas** [comment]
**Two discussions in this ticket:** 

_First Discussion_
The P5 discussion on milestones at http://www.tei-c.org/release/doc/tei-p5-doc/en/html/CO.html#CORS5 doesn't provide any specifics on where to place `<milestone>-like` elements when they appear at division boundaries. Some of the examples imply that they may be outside the lowest level mixed content element, but there is no explicit information.

The reasons suggested for putting the `<milestone>-like` element inside the mixed content element seem to be that it makes it easier to handle links to facsimile images, BPTL is currently in favor of a) putting `<milestone>-like` elements inside the lowest level `<div>` or documenting divergent practice in an `<editorialDecl>`. For the record, Syd and Elli disagree, and think it should be between the sibling sections and not within them. 

_Second discussion:_
P5 doesn't seem to have any explicit advice on how to mark characters like a row of asterisks that indicate some kind of division, but not necessarily a well understood one like a paragraph or a chapter. Epidoc uses `<milestone rend="xx">`. The WWP usually declares it to be marking an identifiable section and uses a `@rend` on a `<div>` or other container element. 
Neither of these is particularly helpful for us in this case. 

It seems to make more sense to treat these characters as content and put them in an `<ab>` 

```
<ab style="xx">*********</ab> 
```

The Epidoc approach is also viable if the separator has a name

```
<milestone rend="paragraphos" unit="undefined"/>
```

Finally, if the separator characters are not in the unicode set and don't have a name, for ex. some kind of curlicue, it  may be possible to use a `<figure>` inside the `<ab>`.

**Comment by emylonas** [comment]
After discussion on 11/13
Explain why we recommend `<ab>` in the prose and add `@style` to example. This is Kevin's original suggestion.

**Comment by emylonas** [comment]
Made changes as above. Note: revisited why not use ``<metamark>``. For the record, it's because that element is part of msDesc and is often used for editorial marks. So ``<ab>`` is definitely better. Closed with commit fbc58b8
